app/assets/javascripts/modules/dashboard.js

```diff
@@ -130,15 +156,19 @@ angular.module('probedock.dashboard', [ 'probedock.api', 'probedock.orgs', 'prob
         return;
       }
 
+      $scope.chart.options.tooltipTemplate = chartConfig[$scope.chart.params.type].tooltipTemplate;
```

Comment:
Not sure if your remark is applicable since the function is called `showMetrics` and not `fetchMetrics`. Can you review this again please.